WtWSMS/history/characters/romano_gallic.txt

```diff
@@ -1,5 +1,5 @@
 ﻿19 = {
-	name = "Flavius_Afranius"	# Rex of Soissons
+	name = "Syagrius"	# Rex of Soissons
```

Comment:
This change doesn't make sense given the way tria nomina worked, the family was the Syagrii

Reply:
> This change doesn't make sense given the way tria nomina worked, the family was the Syagrii

I haven't implemented that in game yet, but I like Settipani's suggestion that Syagrii directly descended from Afranius Hannibalianus (consul of 292) and Publius Afranius Flavianus (suffect consul of 117). I plan to make Syagrii a cadet branch of Afranii when I expand this family.

Reply:
Ok but in the meantime there is a problem because the Syagrii are effectively removed. We had already fixed this previously, I don't see why we should go back to the previous setup.

Reply:
Now both Syagrii and Afranii are represented. I made Syagrii a cadet branch of Flavii Titiani, and their name is transmitted via matrilineal marriage with the Afranii.